src/languages/es-ES.ts

```diff
@@ -2260,6 +2266,37 @@ export default class extends Language {
 			extendedHelp: `This command shows a random starred message or the starboard usage and leaderboard for this server.`
 		}),
 
+		/**
+		 * ####################
+		 * SUGGESTIONS COMMANDS
+		 */
+
+		COMMAND_SUGGEST_DESCRIPTION: 'Manda una sugerencia para el servidor.',
+		COMMAND_SUGGEST_EXTENDED: builder.display('suggest', {
+			extendedHelp: 'Publica una recomendación en el canal de recomendaciones del servidor',
+			explainedUsage: [
+				['suggestion', 'Su recomendación']
+			],
+			examples: [
+				'Crear un canal de música.'
+			],
+			reminder: 'Necesita tener configurado un canal de sugerencias para que este comando funcione. Si eres un administrador, te preguntaré si quieres configurarlo al momento de su uso.'
```

Comment:
```suggestion
			reminder: 'Necesita sugerencias de configuración de canales para que este comando funcione. Si es administrador, se le dará la opción de hacerlo al invocar el comando.'
```
or whatever your B2 Spanish says about the translation

Reply:
I mean... I'd like a second eye on this from @kyranet. It sounds wrong to me

Reply:
kyra said this is fine, so applying